capstone-frontend/src/components/ListPage.jsx

```diff
@@ -77,33 +80,53 @@ class ListPage extends Component {
       :
       (this.state.empty
         ?
-        (<h1 className="text-center mt-4">Booklist has No Books</h1>)
+        (
+          <div>
+            <h2 className="mt-3">{this.state.bookList.name}</h2>
+            <hr style={{ border: "1px solid #ccc" }} />
+            <h3 className="text-center mt-4">Booklist has No Books</h3>
+          </div>
+        )
         : (
-          <div className="text-center mt-4">
-            {
-              this.state.gBooks.map(gBook =>
-                <div key={gBook.id + this.props.match.params.id}>
-                  <a className="text-decoration-none text-body" href={gBook.canonicalVolumeLink}>
-                    <div>
-                      <img src={gBook.thumbnailLink} alt={gBook.title} />
+          <div>
+            <h2 className="mt-3">{this.state.bookList.name}</h2>
+            <hr style={{ border: "1px solid #ccc" }} />
+
+            <div>
+              {
+                this.state.gBooks.map(gBook =>
+                  <Row className="text-center border m-5 bg-light" key={gBook.id + this.props.match.params.id} style={{ border: "1px solid #ccc" }}>
+                    <Col md={3} className="my-4 p-0 ">
+                      {/* TODO(#79): Redirect user to BookPage instead of playstore */}
+                      <a className="text-decoration-none text-body" href={gBook.canonicalVolumeLink}>
+                        <img className="img-responsive" src={gBook.thumbnailLink} alt={gBook.title} />
+                      </a>
+                    </Col>
+
+                    <Col className="my-4 p-0">
+                      <h2 className="mt-4"> {gBook.title} </h2>
+                      <StarRatings
+                        rating={gBook.avgRating}
+                        starDimension="40px"
+                        starSpacing="10px"
+                        starRatedColor="gold" />
+                        <p className="my-3" > {gBook.authors.join(', ')} </p>
+                    </Col>
+
+                    <Col md={3} className="my-4 p-0">
+                      <a className="btn btn-primary mt-5" href={gBook.webReaderLink}>Web Reader</a>
```

Comment:
Can you add a style to this `a` element that will extend it to the full length of the row (`width: 100%`)? I think it looks more consistent if we have all of our buttons the same length and not just the length of their text

Reply:
This ```a``` tag is used as a button that redirects you the web reader.